notebooks/testing_surface_attributes.ipynb

```diff
@@ -0,0 +1,551 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Testing surface attribute calculation\n",
+        "## Introduction\n",
+        "There are different methods available to calculate both gradient (slope steepness and slope direction) and curvature attributes of a given surface.\n",
+        "\n",
+        "The EIS-toolkit contains following defaults, however, they're interchangeable to some point. Results of each calculation returns a dictionary containing the names (key) and respective arrays and metadata (tuple).\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Examples"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### Set-up"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# Modules and basic set-up\n",
+        "import numpy as np\n",
+        "import os\n",
+        "import rasterio\n",
+        "\n",
+        "import sys\n",
+        "sys.path.append('..')\n",
+        "\n",
+        "from eis_toolkit.surface_attributes.parameters import first_order, second_order_basic_set\n",
+        "from eis_toolkit.surface_attributes.utilities import classify_aspect, nodata_to_nan\n",
+        "from eis_toolkit.utilities.conversions import convert_deg_to_rad, convert_rad_to_deg, convert_deg_to_rise\n",
```

Comment:
The imports here were wrong. Correct are I believe as such:

~~~python
from eis_toolkit.utilities.nodata import nodata_to_nan
from eis_toolkit.surface_attributes.classification import classify_aspect
~~~